data/alpha/register/registration-district.yaml

```diff
@@ -7,4 +7,4 @@ fields:
 phase: alpha
 register: registration-district
 registry: hm-passport-office
-text: Registration districts in England and Wales, defined for the purposes of the Registration Acts
+text: Registration districts in England and Wales, defined for the purposes of the Civil Registrations Acts
```

Comment:
Good points, next time I'll include a snippet of the relevant emails.  Maybe we should use GitHub issues, but that becomes yet another channel to track.